pkg/virt-controller/services/generated_mock_vm.go

```diff
@@ -103,30 +102,31 @@ func (_mr *_MockVMServiceRecorder) UpdateMigration(arg0 interface{}) *gomock.Cal
 	return _mr.mock.ctrl.RecordCall(_mr.mock, "UpdateMigration", arg0)
 }
 
-func (_m *MockVMService) FetchVM(vmName string) (*v11.VM, error) {
+func (_m *MockVMService) FetchVM(vmName string) (*v10.VM, bool, error) {
 	ret := _m.ctrl.Call(_m, "FetchVM", vmName)
-	ret0, _ := ret[0].(*v11.VM)
-	ret1, _ := ret[1].(error)
-	return ret0, ret1
+	ret0, _ := ret[0].(*v10.VM)
+	ret1, _ := ret[1].(bool)
+	ret2, _ := ret[2].(error)
+	return ret0, ret1, ret2
 }
 
 func (_mr *_MockVMServiceRecorder) FetchVM(arg0 interface{}) *gomock.Call {
 	return _mr.mock.ctrl.RecordCall(_mr.mock, "FetchVM", arg0)
 }
 
-func (_m *MockVMService) StartMigration(vm *v11.VM, sourceNode *v1.Node, targetNode *v1.Node) error {
-	ret := _m.ctrl.Call(_m, "StartMigration", vm, sourceNode, targetNode)
+func (_m *MockVMService) StartMigration(migration *v10.Migration, vm *v10.VM, sourceNode *v1.Node, targetNode *v1.Node) error {
```

Comment:
Right, but this is generated code by gomock. It decides itself how to alias the imports.